modules/nw-networkpolicy-audit-concept.adoc

```diff
@@ -9,7 +9,14 @@
 You can configure the destination for audit logs, such as a syslog server or a UNIX domain socket.
 Regardless of any additional configuration, an audit log is always saved to `/var/log/ovn/acl-audit-log.log` on each OVN-Kubernetes pod in the cluster.
 
-Audit logging is enabled per namespace by annotating the namespace with the `k8s.ovn.org/acl-logging` key as in the following example:
+You can enable audit logging for each namespace by annotating each namespace configuration with a `k8s.ovn.org/acl-logging` section. In this section, you must specify `allow`, `deny`, or both values to enable audit logging for a namespace.
+
+[NOTE]
+====
+Some vendor networks might use a `pass` action instead of an `allow` action for an ACL rule. Check the vendor documentation to determine the valid value.
```

Comment:
I am not sure of which vendor are we referring here, never tested. We are explaining ovnk ACL audit message logging. Network Policy messages that are enabled with annotation on namespace do not have 'pass' action. But if we are referring to admin network policy (TP feature 4.14 and 4.15, GA in 4.16) it has pass action but does not replace allow action.

Reply:
:heavy_check_mark: